client/webserver/site/src/js/markets.js

```diff
@@ -366,7 +406,7 @@ export default class MarketsPage extends BasePage {
     const orderRows = this.orderRows
     const market = this.market
     for (const oid in orderRows) delete orderRows[oid]
-    const orders = app.orders(market.dex.url, market.base.id, market.quote.id)
+    const orders = app.orders(market.dex.url, market.baseCfg.id, market.quoteCfg.id)
```

Comment:
Orders can only be placed for supported assets, so this change doesn't really do anything.

Reply:
Withouth this change, it will throw an error, even though not being able to send orders.

![image](https://user-images.githubusercontent.com/15069783/81322978-15cedb80-906b-11ea-87e7-51c1005fce00.png)

Reply:
Oh. That's actually just a test app server thing. The fake order creation doesn't check whether assets are supported. You can leave it for now.